examples/getreadme/main.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
```

Comment:
Glad to, though I don't know if I follow the definition of inline. Would it also go `github/repos_test.go`? Happy to have it land where you'd like it to live.